### PR TITLE
Fix tough-cookie Cookie field typings

### PR DESCRIPTION
**Pull request by michaelcypher**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/salesforce/tough-cookie/blob/master/lib/cookie.js#L722
- [x] Increase the version number in the header if appropriate. (**Not appropriate**)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. (**Not making substantial changes**)

### CODE REVIEW

**Comment by mikeecb** [comment]
Hi all, I have no idea why this test is failing (see error below). Any help would be much appreciated.

```
Error in jsdom
Error: Expected compilerOptions["baseUrl"] === "../"
    at Object.<anonymous> (/Users/mcypher/Documents/DefinitelyTyped/node_modules/dtslint/bin/checks.js:72:27)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/mcypher/Documents/DefinitelyTyped/node_modules/dtslint/bin/checks.js:4:58)
    at <anonymous>
Error: The following packages had errors: jsdom
    at doRunTests (/Users/mcypher/Documents/DefinitelyTyped/node_modules/types-publisher/src/tester/test-runner.ts:131:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

**Comment by mikeecb** [comment]
**Update**: Travis CI build has passed.

File: types/jsdom/jsdom-tests.ts [hunk 1]
```diff
@@ -56,8 +56,9 @@ function test_cookieJar() {
     const store = {} as MemoryCookieStore;
     const options = {} as ToughCookieJar.Options;
 
-    const cookieJar = new CookieJar(store, options);
-    const dom = new JSDOM(``, { cookieJar });
+    const cookieJar: CookieJar = new CookieJar(store, options);
+    const constructorOptions: ConstructorOptions = { cookieJar };
+    const dom = new JSDOM(``, constructorOptions);
```
Comment (mikeecb) [inline types/jsdom/jsdom-tests.ts]: I had to change this as the `noImplicitAny` TypeScript check was failing.

**Comment by DanielRosenwasser** [comment]
Any idea what the any message was on, and what it was?

**Comment by DanielRosenwasser** [comment]
Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

**Comment by mikeecb** [comment]
@DanielRosenwasser thanks for the heads up! I've rebased and changed the author so it the email used is associated with my GitHub account. Thanks for reviewing.

**Pull request merged**